modules/bitgo/test/v2/unit/coins/ofcToken.ts

```diff
@@ -284,6 +284,27 @@ describe('OFC:', function () {
     });
   });
 
+  describe('check ofc tokens for arbethErc20', function () {
```

Comment:
Add a `should.equal` for `addressCoin`, since that is a field that we (Prime) uses. Dont want it changing from underneath us without anyone noticing

Reply:
@drew-bitgo added the check. But have to add addressCoin in the ofcTokenConfig as it wasn't there. Please review.